### DISCUSSION TITLE
CLI gives the wrong ruleset name for: "wrapped" and "constrictor".

### DISCUSSION THREAD

**Discussion by EbbDrop** [BattlesnakeOfficial/feedback#161] [Bug Reports]
"wrapped" and "constrictor" default to "standard" in the json request. The rulesets are correctly applied to the game.

Run `battlesnake play -g wrapped -n x -u https://0.0.0.0` to reproduce. You can see the wrong ruleset name in the outputted json for the failed request.

**Comment by bvanvugt** [comment]
Good catch! Saw you put up a PR for this, we'll get it reviewed today. Thanks!

**Reply by bvanvugt** [reply]
Fixed by @EbbDrop in https://github.com/BattlesnakeOfficial/rules/pull/61
